src/resources/Catalogs/Interfaces.ts

```diff
@@ -0,0 +1,30 @@
+export interface CatalogsListOptions {
+    page?: number;
+    pageSize?: number;
+}
+
+export interface CatalogModel {
```

Comment:
Since the `Catalog` can be created without an `Id`, I would suggest creating:

`type CreateCatalogModel = Omit<CatalogModel, 'id'>`

Reply:
By doing this I could set `id` as mandatory in `CatalogModel` right?

Reply:
Exactly :)